src/sveltekit/sdk-setup.ts

```diff
@@ -415,28 +415,95 @@ async function modifyViteConfig(
     return;
   }
 
-  const { org, project, url: sentryUrl, selfHosted } = projectInfo;
+  const { org, project, url, selfHosted } = projectInfo;
 
-  const viteModule = parseModule(viteConfigContent);
-
-  addVitePlugin(viteModule, {
-    imported: 'sentrySvelteKit',
-    from: '@sentry/sveltekit',
-    constructor: 'sentrySvelteKit',
-    options: {
-      sourceMapsUploadOptions: {
-        org,
-        project,
-        ...(selfHosted && { url: sentryUrl }),
+  try {
+    const viteModule = parseModule(viteConfigContent);
+
+    addVitePlugin(viteModule, {
+      imported: 'sentrySvelteKit',
+      from: '@sentry/sveltekit',
+      constructor: 'sentrySvelteKit',
+      options: {
+        sourceMapsUploadOptions: {
+          org,
+          project,
+          ...(selfHosted && { url }),
+        },
       },
-    },
-    index: 0,
-  });
+      index: 0,
+    });
+
+    const code = generateCode(viteModule.$ast).code;
+
+    await fs.promises.writeFile(viteConfigPath, code);
+  } catch (e) {
+    // TODO: log e if --debug is set?
```

Comment:
Will take care of this in a follow-up PR, as we need to bring the `--debug` flag into the clack-based wizard options first

Reply:
#381